### PR TITLE
Implemented the Dashboard Menu.

### PR DESCRIPTION
**Pull request by Skorpios604**

### CODE REVIEW

File: Client/src/Components/DashboardMenu/index.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import language from '@mui/icons-material/language';
+import report_gmailerrorred from '@mui/icons-material/reportGmailerrorred';
+import sensors from '@mui/icons-material/sensors';
+import settings from '@mui/icons-material/settings';
+import all_inclusive from '@mui/icons-material/AllInclusive';
```
Comment (popnfresh234) [inline Client/src/Components/DashboardMenu/index.jsx:5]: These import statements don't work on my end.

I had a look in the documentation, and for the sensors icon for example it looks like it should be:

`import SensorsIcon from '@mui/icons-material/Sensors';`

Can you double check these?

File: Client/src/Components/DashboardMenu/index.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import SvgIcon from '@mui/material/SvgIcon';
+import './index.css'; 
+
+/**
+ * DashboardMenu component displays a menu with icons and corresponding text.
+ * Each menu item consists of an icon and a text label.
+ * 
+ * @returns {JSX.Element} The JSX element representing the DashboardMenu component.
```
Comment (popnfresh234) [inline Client/src/Components/DashboardMenu/index.jsx:13]: You can add the `@component` tag here to let the JSdoc know that this is a react component.  Helps the editor classify things properly.

**Review by popnfresh234 — CHANGES_REQUESTED** [review]
Something strange going on with the icon imports, can you double check your packages and import statements?

File: Client/src/Components/DashboardMenu/index.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import LanguageIcon from "@mui/icons-material/Language";
+import ReportGmailerrorredIcon from "@mui/icons-material/ReportGmailerrorred";
+import SensorsIcon from "@mui/icons-material/Sensors";
+import SettingsIcon from "@mui/icons-material/Settings";
+import AllInclusiveIcon from "@mui/icons-material/AllInclusive";
+import SvgIcon from '@mui/material/SvgIcon';
+import './index.css';
```
Comment (popnfresh234) [inline Client/src/Components/DashboardMenu/index.jsx:7]: Nice, imports are good, everything compiling and rendering nicely.  Thanks for making the change, still no idea why it was working on your end :joy:   All's well that ends well.

Comment (Skorpios604) [reply Client/src/Components/DashboardMenu/index.jsx:7]: No worries! Thanks for the review. Product is ready for deployment 🚀 lol!

**Pull request merged**